pkg/apis/v1alpha1/command.go

```diff
@@ -0,0 +1,19 @@
+package v1alpha1
+
+// Command describes a command to run as a part of a test step or suite.
+type Command struct {
+	// The command and argument to run as a string.
+	Command string `json:"command,omitempty"`
```

Comment:
Why `omitempty` ?

Reply:
there could be a command or the script 
so I think having a command string is not necessary always

Reply:
Hmm, right. How is the script processed ? 🤔

Reply:
we just do it like : 
command = exec.CommandContext(ctx, "sh", "-c", cmdStr)

Reply:
Ok so it requires a shell...

Reply:
We typically build docker images that don't have a shell 🤷

Reply:
yes

Reply:
> We typically build docker images that don't have a shell 🤷

but how would I exec a shell command now.